middlewares/auth/verifyIdToken.js

```diff
@@ -0,0 +1,35 @@
+const HttpError = require("http-errors");
+const { OAuth2Client } = require("google-auth-library");
+
+const User = require("../../models/User");
+const createHttpError = require("../../utils/createHttpError");
+const { LOGIN } = require("../../constants/error");
+
+const verifyIdToken = async (req, res, next) => {
+  const { authorization, "client-id": clientId } = req.headers;
```

Comment:
```suggestion
  const { Authorization, "client-id": clientId } = req.headers;
```
대문자로 시작 하는 걸로 알고 있습니다